docs/download_files.md

```diff
@@ -7,6 +7,10 @@ The folder structure within the zip file is determined by whether individual sam
 ## Download folder structure for project downloads:
 ![docs-downloads-project](https://user-images.githubusercontent.com/15315514/143308420-a3cca10d-814f-4c52-b934-98d5e9cef1c5.png)
 
+If a project contains bulk RNA-sequencing samples, a single tab-separated values file, `SCPCP000000_bulk_quant.tsv` will be included in the download. 
+The contents of this file include a gene by sample matrix containing raw gene expression counts data determined using Salmon.
```

Comment:
This file does not have anything but the gene by sample matrix, correct? In my mind "contents include" implies that there may be other things as well.

I also clarified the row/column contents, but I don't know if that is necessary.
```suggestion
This file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
```